include/pros/apix.h

```diff
@@ -307,9 +307,10 @@ bool queue_peek(queue_t queue, void* const buffer, uint32_t timeout);
  * \param buffer
  *        Pointer to a buffer to which the received item will be copied
  * \param timeout
- *        Time to wait for space to become available. A timeout of 0 can be used
- *        to attempt to post without blocking. TIMEOUT_MAX can be used to block
- *        indefinitely.
+ *        The maximum amount of time the task should block
+ *        waiting for an item to receive should the queue be empty at the time
+ *        of the call. queue_recv() will return immediately if timeout
+ *        is zero and the queue is empty.
```

Comment:
```suggestion
 *        of the call. queue_recv() will return immediately if the timeout
 *        is zero.
```

I think it would also be nice to keep the mention to TIMEOUT_MAX, but ¯\\\_(ツ)_/¯

Reply:
This is copied from the freeRTOS doc comment, there is a chance this blocks if the queue is not empty, it will yield to allow all the waiting publishers to publish before even checking the timeout.